docs/porting_guide.md

```diff
@@ -28,11 +28,11 @@ platform/
 │   └── sai_npu.py    (optional)
 ```
 
-The platform folder MAY contain multiple vendors. Each vendor folder MAY contain 1..n ASICs and optional vendor-specific `sai_npu` module. Each ASIC folder MAY contain 1..n targets (either HW device or simulator or emulator) and optional ASIC-specific `sai_npu` module. Each target's folder MUST contain either `Dockerfile` for standalone mode or `Dockerfile.server` for client-server mode or both. Also, the target's folder MAY contain optional `sku/` folder with JSON files that define front panel ports configuration as well as others target-specific files and folders.
+The NPY folder ***may*** contain multiple vendors. Each vendor folder ***may*** contain 1..n ASICs and optional vendor-specific `sai_npu` module. Each ASIC folder ***may*** contain 1..n targets (either HW device or simulator or emulator) and optional ASIC-specific `sai_npu` module. Each target's folder ***must*** contain either `Dockerfile` for standalone mode or `Dockerfile.server` for client-server mode or both. Also, the target's folder ***may*** contain optional `sku/` folder with JSON files that define front panel ports configuration as well as others target-specific files and folders.
```

Comment:
The idea was to follow RFC style - "MUST", "MAY", etc. Please revert.

Reply:
Fixed